mlflow/langchain/utils/chat.py

```diff
@@ -213,7 +213,13 @@ def _convert(chunk):
     return map(_convert, chunk_iter)
 
 
-def _convert_chat_request_or_throw(chat_request: dict):
+def _convert_chat_request_to_messages_or_throw(chat_request: dict) -> list[BaseMessage]:
```

Comment:
when does this function throw?

Reply:
When convert_to_messages fails, e.g. the dictionary doesn't follow {"role": ..., "content": ...} format, or the role is not supported